maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -184,26 +176,141 @@ which is suitable:
     ```bash
     kubectl apply -f {{ "/manifests/calico-vxlan.yaml" | absolute_url }}
     ```
-    
-  * Restart any pods that were created before {{site.prodname}} was installed.  Typically, this is only `kube-dns`:
-  
-    ```
-    kubectl delete pod -n kube-system -l k8s-app=kube-dns
-    ```
+
+* Create a nodegroup, using the AMI ID you noted above.
+  * `--node-ami` should be set to the AMI ID of the image built above.
+  * `--node-ami-family` should be set to `Ubuntu1804` (despite the upgrade).
+
+  For example:
+  ```
+  eksctl create nodegroup \
+    --cluster my-calico-cluster \
+    --node-type t3.medium \
+    --node-ami auto \
+    --max-pods-per-node 100 \
+    --node-ami-family Ubuntu1804 \
+    --node-ami <AMI ID>
+  ```
 
 %>
 {% endtabs %}
 
-* Continue with the instructions in the main [Enabling eBPF page](./enabling-bpf).  
-
-  When configuring {{site.prodname}} to connect to the API server, use the load balanced domain name created by EKS.
-  One way to determine the domain name is to extract it from `kube-proxy`'s config map:
+#### Configure {{site.prodname}} to connect directly to the API server
```

Comment:
Configure {{site.prodname}} to connect to the API server

Reply:
"Directly" is important here - before kube-proxy is disabled, calico-node can already connect to the API server via the service IP set up by kube-proxy.  

Once we delete kube-proxy, calico-node (and typha, calico-kube-controllers) will need to connect to the API server directly - i.e. without any assistance from kube-proxy.

Reply:
Got it. Ignore